lib/parse.js

```diff
@@ -10,7 +10,13 @@
 module.exports = function (input, callback) {
     var result;
     try {
-        result = JSON.parse(input);
+        // The input is a JSON string, which may contain control characters
+        // that should be removed. See LLK/scratch-vm#1077
+        // So far we've only encountered the backspace control character,
+        // so remove that specific one before continuing.
+        // SB2 JSONs and SB3 JSONs have different versions of the
+        // character serialized (e.g. \u0008 and \b), strip out both versions
+        result = JSON.parse(input.replace(/\\b|\\u0008/g, ''));
```

Comment:
Would this make backspace characters intentionally used as the input to a "key () pressed?" block break, because the input would be empty (and hence wouldn't detect backspace, as it was intended)?

Reply:
Yes, this would remove backspace control characters that are intentionally added (via "hacked" arguments).

Reply:
In that case, we would be "fixing" those projects in that they would now load, but they would in fact not operate the way they're meant to, correct? I understand that this PR also fixes issues with JP keyboard input, but does that outweigh the still half-broken projects this PR would cause?

Reply:
Question: if this PR is merged, and a project owner opens their (broken) project and saves it, will the backspace character be removed from the saved JSON? If so, we'd be breaking those projects irreversibly, when there might be solutions that could keep the backspace character working. 

This might not be a bad thing as long as auto-save doesn't automatically trigger when the project owner looks inside their project. Does this happen?